### DISCUSSION TITLE
[Question] Osclass 5.beta9

### DISCUSSION THREAD

**Discussion by eurobank** [mindstellar/shopclass#332] [Q&A] [answered]
Is that changed? In previous beta it was different? Not sure 100%.

![Screenshot](https://user-images.githubusercontent.com/4467158/122974775-5f77a400-d39b-11eb-91e7-d9be1cfe00da.png)

**Comment by navjottomer** [comment, marked as answer]
If a maintenance template is available in the current theme then it will be used as a maintenance template. 
Was added months ago.
